apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/test/get-editor-type.test.ts

```diff
@@ -0,0 +1,36 @@
+import { getEditorType } from '../get-editor-type';
+
+jest.mock( '@wordpress/data', () => {
+	return {
+		select: jest.fn(),
+	};
+} );
+
+const { select } = jest.requireMock( '@wordpress/data' );
+
+describe( 'getEditorType', () => {
+	it( 'should return `post`', () => {
+		( select as jest.Mock ).mockImplementationOnce( () => {
+			return {
+				getCurrentPostType: () => 'post',
+			};
+		} );
+		expect( getEditorType() ).toEqual( 'post' );
+	} );
+	it( 'should return `page`', () => {
+		( select as jest.Mock ).mockImplementationOnce( () => {
+			return {
+				getCurrentPostType: () => 'page',
+			};
+		} );
+		expect( getEditorType() ).toEqual( 'page' );
+	} );
+	it( 'should return `site` when the post type is `null`', () => {
```

Comment:
Changed the implementation by [this](https://github.com/Automattic/wp-calypso/pull/73800#issuecomment-1449349064). 
I suppose we don't need tests here since we prefer to see a stable utility on the Gutenberg side. https://github.com/Automattic/wp-calypso/pull/73800/commits/3fa59ec3f58a36bf6ed44486df5b01a96068fb46